plugins/org.eclipse.glsp.server/src/org/eclipse/glsp/server/features/core/model/ModelSubmissionHandler.java

```diff
@@ -36,7 +36,7 @@
 @Singleton
 public class ModelSubmissionHandler {
 
-   @Inject()
+   @Inject
```

Comment:
I'm in favor of removing unnecessary empty `()` from inject calls. But if we do that it should be done consistently.
I did a quick search and it seems like we are using `inject()` in multiple other places as well